templates/index.html

```diff
@@ -32,362 +30,475 @@
-                    id="Finance"
-                    tabindex="-1">Finance</button>
+      <!-- lottie animation -->
+      <div class="u-hide--small u-hide--medium">
+        <div id="centre-animation-wrapper">
+          <div class="centre-animation__zones" aria-label="Features of ubuntu pro">
+            <div class="centre-animation__zone--1"
+                 aria-label="Hardening and certifications"></div>
+            <div class="centre-animation__zone--2"
+                 aria-label="Reliable security maintenance"></div>
+            <div class="centre-animation__zone--3" aria-label="Simplified compliance"></div>
+            <div class="centre-animation__zone--4"
+                 aria-label="Patching automation tools"></div>
+            <div class="centre-animation__zone--5" aria-label="24/7 Support"></div>
+            <div class="centre-animation__zone--6"
+                 aria-label="Long-term API stability"></div>
+            <div class="centre-animation__zone--7" aria-label="Device management"></div>
           </div>
+          <div id="centre-animation" style="height: auto; width: 100%"></div>
         </div>
       </div>
-      <div tabindex="0" role="tabpanel" id="Telco-tab" aria-labelledby="Telco">
-        <div class="p-section">
-          <div class="row">
-            <div class="col-2 col-medium-2 col-small-2">
-              {{ image(url="https://assets.ubuntu.com/v1/dd7614e3-BT%20logo.png",
-                            alt="",
-                            width="1196",
-                            height="577",
-                            hi_def=True,
-                            loading="lazy") | safe
-              }}
-            </div>
-            <div class="col-9 col-start-large-4 col-medium-4 col-small-4">
-              <div class="row p-section--shallow">
-                <div class="col-6 col-medium-4">
-                  <h3 class="p-heading--2">Next-generation 5G networks</h3>
-                </div>
-                <div class="col-3 col-medium-4">
-                  <p>BT deploys and scales a responsive 5G network with Canonical.</p>
-                  <p>
-                    <a href="https://ubuntu.com/blog/bt-turns-to-canonical-ubuntu-to-enable-next-generation-5g-cloud-core">Read
-                    the press release&nbsp;›</a>
-                  </p>
-                </div>
-              </div>
-              {{ image(url="https://assets.ubuntu.com/v1/41ac89a1-GettyImages-836184268.jpg",
-                            alt="",
-                            width="1880",
-                            height="699",
-                            hi_def=True,
-                            loading="lazy") | safe
-              }}
-            </div>
-          </div>
+    </div>
+  </section>
+
+  <section class="p-section--deep">
+    <div class="row">
+      <div class="col-9 col-start-large-4">
+        <div class="p-section--shallow">
+          <h2>Do more with trusted open source</h2>
```

Comment:
This section seems misaligned starting at the following breakpoint:
![image](https://github.com/user-attachments/assets/bec223a1-c312-4686-8193-cf320448db14)

Looks fine on mobile though:
![image](https://github.com/user-attachments/assets/795f7cf9-cc73-4050-8e42-a6dab231bf3f)